src/app/forms/custom-sections/tyres/tyres.component.ts

```diff
@@ -21,34 +28,122 @@ export class TyresComponent implements OnInit, OnDestroy, OnChanges {
   public form!: CustomFormGroup;
   private _formSubscription = new Subscription();
 
-  constructor(public dfs: DynamicFormService) {}
+  constructor(public dfs: DynamicFormService, private referenceDataService: ReferenceDataService) {}
 
   ngOnInit(): void {
-    const template = getTyresSection(this.vehicleTechRecord.vehicleType);
+    this.form = this.dfs.createForm(this.template, this.vehicleTechRecord) as CustomFormGroup;
+    this._formSubscription = this.form.cleanValueChanges.pipe(debounceTime(400)).subscribe(event => {
+      this.formChange.emit(event);
+    });
```

Comment:
Don't know if the formatting change was done on purpose 
```suggestion
    this._formSubscription = this.form.cleanValueChanges.pipe(debounceTime(400)).subscribe(event => this.formChange.emit(event));
```